SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -344,6 +344,19 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	static bool IsRegionCodeValid(const ERegionCode::Type RegionCode);
 	static bool IsManualWorkerConnectionSet(const FString& LaunchConfigPath);
 
+public:
+	UPROPERTY(EditAnywhere, config, Category = "Mobile", meta = (DisplayName = "Connect to a local deployment"))
+		bool bMobileConnectToLocalDeployment;
+
+	UPROPERTY(EditAnywhere, config, Category = "Mobile", meta = (DisplayName = "Mobile Client Worker Type"))
+		FString MobileWorkerType = SpatialConstants::DefaultClientWorkerType.ToString();
+
+	UPROPERTY(EditAnywhere, config, Category = "Mobile", meta = (EditCondition = "bMobileConnectToLocalDeployment", DisplayName = "Runtime IP to local deployment"))
```

Comment:
Could you set a default value `127.0.0.1` to the field?

Reply:
Could do. but that will never work for mobile as you can't connect to localhost SpatialOS. So would prefer to keep it in empty so that people are forced to enter some kind of value